src/arrays.js

```diff
@@ -1,61 +1,126 @@
 const getNthElement = (index, array) => {
   // your code here
+  if (index > array.length - 1) {
+    index -= array.length;
+  }
+  return array[index];
 };
 
 const arrayToCSVString = array => {
   // your code here
+  return array.reduce((total, element) => `${total},${element}`);
 };
 
 const csvStringToArray = string => {
   // your code here
+  const newArr = [];
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
   // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
   // your code here
+  const secondArr = [element];
+  return array.concat(secondArr);
 };
 
 const removeNthElement = (index, array) => {
   // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
   // your code here
+  return numbers.map(number => {
+    return number.toString();
+  });
 };
 
 const uppercaseWordsInArray = strings => {
   // your code here
+  return strings.map(element => {
+    return element.toUpperCase();
+  });
 };
 
 const reverseWordsInArray = strings => {
   // your code here
+  // 1. map() through 'strings' array to reach each word one by one so you can do things to them in turn (reverse letters)
+  // 2. for each word, turn it into an array of characters
+  const arrayOfArraysOfCharacters = strings.map(string => string.split(''));
+  // 3. map through the new array of arrays and reverse() each array of characters
+  const arrayOfArraysOfReversedCharacters = arrayOfArraysOfCharacters.map(
+    array => array.reverse()
+  );
+  // 4. map through the new array and reduce each array of characters to a string (you get the reversed words into a new array)
+  const finalArrayOfReversedWords = arrayOfArraysOfReversedCharacters.map(
+    array => {
+      return array.reduce((reversedWord, currentLetter) => {
+        return reversedWord + currentLetter;
+      });
+    }
+  );
+  return finalArrayOfReversedWords;
```

Comment:
Theres a lot of noise here in the comments that doesnt need to be. It looks like you're trying to explain _how_ you're doing something rather than _why_, which is more valuable for whoever will come to read this in the future. As these are standard methods it would be easy enough to look up what they are doing if the reader is unsure, so these comments should all be removed to make it easier to read.
While this works, it is a bit over complicated, and as it happens in this case the logic can be greatly simplified using almost the same methods you have but with a bit of refactoring:

```suggestion
  return strings.map(string =>
    string
      .split('')
      .reverse()
      .join('')
  );
```